report/src/main/webapp/package-lock.json

```diff
@@ -0,0 +1,13796 @@
+{
```

Comment:
This file looks like generated one. Do we need this here?

Reply:
Yes, my bad. Gonna get rid of it.